SFDigitalServices/sf-dahlia-lap
# [DAHLIAFE-26] Update tab and card styling

https://sfgovdt.jira.com/browse/DAHLIAFE-26

Pattern lib PR: https://github.com/SFDigitalServices/sf-dahlia-pattern-library/pull/139

### This PR
1. Updates tabs to match figma and look reasonable for medium + small screen sizes
1. Removes the "App card" border container around things to get the full horizontal white line
1. Makes currency and percent fields have a border radius that matches other fields

### This PR does not
1. Make any changes to general padding/styling of form fields -> I'm assuming that that'll make the most sense to do when we re-arrange other fields


### Pieces to test
1. Make sure tabs still work
1. See tabs on both supp app + listing page (where you click back and forth between listing + applications)
